rnr_control/src/rnr_control.c

```diff
@@ -174,8 +249,9 @@ static void handle_arguments( ps_node_ref const node_ref, int argc, char **argv
     unsigned int quit_session = 0;
     unsigned int enumerate_all_sessions = 0;
     unsigned int enumerate_local_sessions = 0;
+    char my_path[100];
```

Comment:
This is prone to bufferoverflows. Update to use the macro `PSYNC_DEFAULT_STRING_LEN`, which is equivalent to `2048 bytes`.